### ISSUE TITLE
[Crash] Game crashes everytime I load a world

### ISSUE THREAD

**Issue by Sollux4Smash** [sp614x/optifine#4996]
## Description of Issue
Every time I load a world on my 1.16.3 modded installation it crashes, and I narrowed it down to the last mod I was expecting, Optifine. The crash still happens if Optifine is the only mod installed, so I want to ask for help here. I've pasted the full crash report below.

## Steps to Reproduce
1. Open the world 
2. Immediate crash

## OptiFine Version
HD U G4 

## Installation Method
Dropped JAR file into mod folder

## Fabric/Forge Version
34.1.42

## Other Installed Mods
None. I had other mods but this crash still happens when I remove everything but Optifine

## Log Files/Crash Reports
https://pastebin.com/H7z3rZyb

## F3 Debug Screenshot
Can't do this because I can't get in-game

## Additional Information
I have tried some fixes I saw for other people posting about this issue for vanilla minecraft, mainly turning of Windows Defender settings for Private networks. I think the issue is some conflict between Optifine and something with my computer, so if there is any information that you need about my computer to help me figure out what to do I can do my best to provide that.

**Comment by arashi002** [comment]
I have the same problem and I did my searched what makes the game crashed and it's just that optifine is not compatible with forge 34.1.42 but it works well with 34.1.0

**Comment by Sollux4Smash** [comment]
> I have the same problem and I did my searched what makes the game crashed and it's just that optifine is not compatible with forge 34.1.42 but it works well with 34.1.0

That worked. It's almost frustrating that the fix was that simple after spending hours trying to figure this out. Thank you so much for the help tho!

**Comment by neos7m** [comment]
Same problem. Can't revert to 34.1.0 because BiomesOPlenty depends on 34.1.33. I opened an issue a few days ago and haven't had any answers yet

**Comment by natsumatsuri** [comment]
> Same problem for me, even our logs match. Can't revert to 34.1.0 because BiomesOPlenty depends on 34.1.33. I opened an issue a few days ago and haven't had any answers yet

I tried Forge 34.1.33 and it works for me, not for you?

**Comment by neos7m** [comment]
> 
> 
> > Same problem for me, even our logs match. Can't revert to 34.1.0 because BiomesOPlenty depends on 34.1.33. I opened an issue a few days ago and haven't had any answers yet
> 
> I tried Forge 34.1.33 and it works for me, not for you?

No, I can confirm it does not work for me even with Forge 34.1.33. I tried both G3 and G4.

**Comment by neos7m** [comment]
For me, it was fixed after upgrading to Optifine 1.16.3 HD U G5. Could you try it and see if it solves it for you?

**Comment by Horos402** [comment]
I have a similar issue, but i do't use forge. the game crashes and says "session.lock is no longer valid. sometimes works deleting session.lock from the world folder but 90% of the time crashes.
